### PR TITLE
[W6.6h][T15-B3]Darren Chin Jhun Pyng

### PR DESCRIPTION
**Pull request by DarrenCzen**
Updated DeveloperGuide.md with rename Tag use case

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -88,6 +88,34 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename a tag
+2. AddressBook requests user to enter name of tag to rename and new name for the tag.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:96]: see if you can split this to multiple steps?
same for step 4.
this is also OK, but splitting would make it more granular

**Comment by okkhoy** [comment]
@DarrenCzen 
Some comments added. Please ack & close the PR after reading comments.

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (DarrenCzen) [reply doc/DeveloperGuide.md:96]: Alright! thanks for the advice!

**Comment by DarrenCzen** [comment]
Alright! thanks for the advice!

**Pull request closed without merging**